Cargo.toml

```diff
@@ -1,4 +1,6 @@
 [workspace]
+rust-version = "1.56.0"
+edition = "2021"
```

Comment:
Is this correct to be added to the `[workspace]` section?

Reply:
no, it has to be added to the individual cargo toml files, working on a fix